README.md

```diff
@@ -1050,7 +1050,7 @@ We provide a [starter template for a custom policy](https://github.com/Polly-Con
 
 Polly now has a [Polly-Contrib](https://github.com/Polly-Contrib) to allow the community to contribute policies or other enhancements around Polly with a low burden of ceremony.
 
-Have a contrib you'd like to publish under Polly-Contrib? Contact us with  an issue here or on [Polly slack](http://pollytalk.slack.com), and we can set up a CI-ready Polly.Contrib repo to which you have full rights, to help you manage and deliver your awesomeness to the community!
+Have a contrib you'd like to publish under Polly-Contrib? Contact us with  an issue here or on [Polly slack](http://pollytalk.slack.com), and we can set up a CI-ready Polly. Contrib repo to which you have full rights, to help you manage and deliver your awesomeness to the community!
```

Comment:
Polly.Contrib is a namespace/assembly name in this context so it was correct as it was.
```suggestion
Have a contrib you'd like to publish under Polly-Contrib? Contact us with  an issue here or on [Polly slack](http://pollytalk.slack.com), and we can set up a CI-ready Polly.Contrib repo to which you have full rights, to help you manage and deliver your awesomeness to the community!
```

Reply:
Makes Sense. Thanks